src/Forte.Optimizely.ContentUsage/Api/Features/ContentUsage/GetContentUsagesQuery.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Forte.EpiContentUsage.Api.Common;
+
+namespace Forte.Optimizely.ContentUsage.Api.Features.ContentUsage;
+
+public class GetContentUsagesQuery
+{
+    public string Query { get; set; }
```

Comment:
`Query` prop is a bit not clear here, especially used in a `...Query` class.
Let's consider renaming it to `NamePhrase`